R/exp_all_figs_tables.R

```diff
@@ -0,0 +1,47 @@
+#' Export all figures and tables
+#'
+#' Export all figures in tables within one function.
+#'
+#' @inheritParams plot_recruitment
+#' @inheritParams afsc_tier_table
+#' @inheritParams plot_biomass
+#' @inheritParams plot_landings
+#' @inheritParams plot_spawning_biomass
+#' @inheritParams table_bnc
+#' @inheritParams table_indices
+#' @inheritParams harvest_projection_table
+#'
+#' @return
+#' Rda files for each figure/table.
+#' @export
+#'
+exp_all_figs_tables <- function(
+    # arguments within plot_recruitment
+    dat,
+    params = FALSE,
+    params_only = FALSE,
+    units = c(sb = "metric tons", recruitment = "metric tons"), # ALSO IN landings
+    recruitment_units = "metric tons",
+    spawning_biomass_units = "metric tons",
+    scaled = FALSE,
+    scale_amount = NULL,
+    show_warnings = FALSE,
+    end_year = NULL,
+    return = "recruitment",
+    make_rda = FALSE,
+    rda_folder = getwd(),
```

Comment:
I suggest changing this parameter name to contain dir to be consistent with other uses for folder directories among the code. This would have to be changed throughout for consistency. Up to you if you feel like this matters much though

Reply:
✔️ will be removed in next commit of add-rda branch